### PR TITLE
Added Starlark support

### PR DESCRIPTION
**Pull request by UebelAndre**
This implements the feature request #158 and adds [starlark](https://github.com/bazelbuild/starlark/blob/master/spec.md) support to this plugin

### CODE REVIEW

**Comment by UebelAndre** [comment]
@NilsJPWerner Hey, thought I'd ping this PR to see if I can get a review from you 🙏

**Comment by NilsJPWerner** [comment]
Hey! I will check it in the next few days!

**Comment by UebelAndre** [comment]
Oh wow!! Thank you so much!! I have very little experience with typescript and vscode extensions so looking forward to your feedback and learning a thing or two 😄

**Comment by UebelAndre** [comment]
I've also noticed that this functionality isn't always available in starlark files. I've found that if I open a python file that it usually works after that. Any suggestions?

**Comment by UebelAndre** [comment]
Hey, just pinging this PR again 😅

**Comment by NilsJPWerner** [comment]
Hi @UebelAndre sorry for the delay. I've been busy finishing up work for the quarter ending tomorrow. I promise I will check this this weekend.

**Comment by UebelAndre** [comment]
Hey, just a friendly ping 😅

File: src/extension.ts [hunk 1]
```diff
@@ -20,33 +20,33 @@ export function activate(context: vs.ExtensionContext): void {
...
                 },
-            },
-            '"',
-            "'",
-            "#",
-        ),
-    );
+                '"',
+                "'",
+                '#',
```
Comment (NilsJPWerner) [inline src/extension.ts:38]: Can you switch back to using double quotes here?

File: package.json [hunk 2]
```diff
@@ -136,9 +137,10 @@
         "vscode:prepublish": "npm run compile",
         "copy_templates": "copyfiles --up 1 \"src/docstring/templates/*.mustache\" out/",
         "copy_python_test_files": "copyfiles --up 1 \"src/test/integration/python_test_files/*\" out/",
+        "copy_starlark_test_files": "copyfiles --up 1 \"src/test/integration/starlark_test_files/*\" out/",
         "compile": "npm run copy_templates && tsc -p ./",
         "watch": "npm run copy_templates && tsc -watch -p ./",
-        "preintegration_test": "npm run copy_python_test_files && npm run compile",
+        "preintegration_test": "npm run copy_python_test_files && npm run copy_starlark_test_files && npm run compile",
```
Comment (NilsJPWerner) [inline package.json:143]: I think I would prefer to just have the .sky files in the python directory and not bother with the additional copy command

**Comment by UebelAndre** [comment]
@NilsJPWerner ready for another review 😄

**Comment by UebelAndre** [comment]
Hey, just pinging this 😅

**Review by NilsJPWerner — APPROVED** [review]

**Comment by UebelAndre** [comment]
Thank you! Can I get a release for this as well? 🙏

**Comment by NilsJPWerner** [comment]
Yeah I'll create a release over the weekend. Thanks for the PR, and sorry for the delay in checking it. It's been a crazy few weeks.

**Comment by UebelAndre** [comment]
Hey! Hope you're having a good weekend 😄 Any updates on a release? 🙏

**Comment by NilsJPWerner** [comment]
0.5.4 has been published now

**Pull request merged**